addons/website/controllers/main.py

```diff
@@ -725,7 +727,8 @@ def hybrid_list(self, search='', search_type='all', **kw):
             return request.render("website.list_hybrid")
 
         options = self._get_hybrid_search_options(**kw)
-        data = self.autocomplete(search_type=search_type, term=search, order='name asc', limit=100, max_nb_chars=200, options=options)
+        limit = 24
```

Comment:
```suggestion
        limit = kw.get('limit') || 24
```
weird assignment :slightly_frowning_face:, let's make it more flexible & structured by using kwArgs too.